### PR TITLE
Fix/country profile

### PR DESCRIPTION
**Pull request by shreeyash07**
- Addresses #49 

## Changes
- Remove Status Card component
- Modify percentageStat Component
- Add Readiness component

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] codegen errors
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments
- [x] conflict markers

### CODE REVIEW

File: app/Base/styles.css [hunk 1]
```diff
@@ -98,13 +98,16 @@
...
 
     --color-foreground: #ffffff;
     --color-background: #fafafa;
     --color-background-dark: #193256;
     --color-text-on-dark: #ffffff;
     --color-text: #333333;
-    --color-text-regional : #2F9C67;
+    --color-text-regional : #8DD2B1;
+    --color-background-readiness: #E8F9F1;
+    --color-readiness-title: #2F9C67;
```
Comment (subinasr) [inline app/Base/styles.css]: Let's keep the variable name generic like `color-background-alt` or `color-background-dark`

File: app/views/Dashboard/Country/index.tsx [hunk 2]
```diff
@@ -17,57 +17,79 @@ import {
...
+const COLORS = ['#567968', '#52625A', '#AFFAD5'];
 
 function Country(props: CountryProps) {
     const {
         className,
     } = props;
 
-    const rendererParams = useCallback((_, data: StatusCardProps) => ({
-        statusId: data.statusId,
+    const rendererParams = useCallback((_, data: PercentageStatsProps) => ({
```
Comment (subinasr) [inline app/views/Dashboard/Country/index.tsx]: Let's name this  properly
```suggestion
    const statusRendererParams = useCallback((_, data: PercentageStatsProps) => ({
```

**Review by subinasr — CHANGES_REQUESTED** [review]

File: app/Base/styles.css (hunk 1 above)
Comment (shreeyash07) [reply app/Base/styles.css]: done

File: app/views/Dashboard/Country/index.tsx (hunk 2 above)
Comment (shreeyash07) [reply app/views/Dashboard/Country/index.tsx]: Done

File: app/views/Dashboard/Country/index.tsx [hunk 3]
```diff
@@ -17,57 +17,79 @@ import {
...
+                                className={styles.readinessListCard}
+                                renderer={ReadinessCard}
+                                rendererParams={readinessRendererParams}
+                                data={readinessData}
+                                keySelector={readinessKeySelector}
+                                errored={false}
+                                filtered={false}
+                                pending={false}
+                            />
+                        </div>
```
Comment (AdityaKhatri) [inline app/views/Dashboard/Country/index.tsx]: remove extra divs if possible

**Review by AdityaKhatri — CHANGES_REQUESTED** [review]

File: app/views/Dashboard/Country/index.tsx (hunk 3 above)
Comment (shreeyash07) [reply app/views/Dashboard/Country/index.tsx]: Done

**Review by subinasr — APPROVED** [review]

**Review by AdityaKhatri — APPROVED** [review]

**Pull request merged**